src/__tests__/integration.test.js

```diff
@@ -45,14 +45,14 @@ describe('Filter tests', () => {
 describe('Pricing tests', () => {
   it('will render zero when unchecked', () => {
     const app = mount(<App />)
-    expect(app.find('#total-price').text()).toBe('0')
+    expect(app.find('#total-price').text()).toBe('Total: $0')
```

Comment:
Your tests didn't need much fixing? It means that you built strong, robust tests or it means that you don't have sufficient test coverage. Not sure which one it is?